types/controller.go

```diff
@@ -140,16 +139,21 @@ func synchronizeLookups(ticker *time.Ticker,
 	lookupBuilder *FunctionLookupBuilder,
 	topicMap *TopicMap) {
 
-	for {
-		<-ticker.C
+	fn := func() {
 		lookups, err := lookupBuilder.Build()
 		if err != nil {
 			log.Fatalln(err)
 		}
-
-		log.Println("Syncing topic map")
```

Comment:
I know you that you are not keen on this message :-) we should probably leave it in though for the time being.

Reply:
Yeah, my concern is that it will flood logs over time for little to no added benefit. 🙂 I've added it back nonetheless.